### PR TITLE
Fixes #28: Make site in design of Fossasia main site

### LINKED ISSUE #16: Adapt colors and design to FOSSASIA standard
The site uses green colors and areas have a number of issues. Please adapt colors and design of the site to use the FOSSASIA standard design.

### LINKED ISSUE #28: Make site in design of fossasia main site.
Please make the site look closer to main site.

### PR DESCRIPTION
**Pull request by Ploypiti**
Fixes #16 #17 #28 
[Preview here](https://ploypiti.github.io/2014.fossasia.org/)

- Change fonts & Color to match the main site
- Change menu and hover color
- Remove shadow from sponsor logo and expand the logo on hover instead
- Fix the header & text that wasn't working
- Change readme.md from `Google code-in site` to `Fossasia summit 2014`

image

### CODE REVIEW

**Review by nzec — COMMENTED** [review]
Woah! Did a lot of work! Please rebase, all the newest changes in the main repository are not present in yours.

**Review by gabru-md — APPROVED** [review]
LGTM

File: _includes/header.html [hunk 1]
```diff
@@ -4,57 +4,48 @@
...
 
 		#side-bar h1 {
-			font-family: 'Lobster';
-			color: #DDD;
+			font-family: 'raleway';
+			color: #333;
+     			 font-size: 1.4em;
+     			 text-transform: none;
+      			 letter-spacing: 0;
+			padding: 0px;
```
Comment (akshatnitd) [inline _includes/header.html]: Please correct the indentation.

File: _includes/header.html [hunk 2]
```diff
@@ -88,13 +84,19 @@
 			width: 60%;
 			height: 30px;
 			margin-top: 5%;
-			transition: 0.5s;
-			-webkit-transition: 0.5s;
-			-moz-transition: 0.5s;
+			background-color: #e12b00;
+      border-radius: 15px;
```
Comment (akshatnitd) [inline _includes/header.html]: Please correct the indentation.

File: _includes/header.html [hunk 3]
```diff
@@ -186,8 +188,9 @@
 }
 
 h2 {
-	font-family: 'Droid Sans';
+	font-family: 'Open Sans';
 	text-align: center;
+  font-weight: 400;
```
Comment (akshatnitd) [inline _includes/header.html]: Please correct the indentation.

File: _includes/header.html [hunk 4]
```diff
@@ -217,33 +220,39 @@
 /*Style for menu links*/
 li a {
 	display:block;
+  font-size: 12px;
```
Comment (akshatnitd) [inline _includes/header.html]: Please correct the indentation.

File: _includes/header.html [hunk 5]
```diff
@@ -217,33 +220,39 @@
...
+	color: #333;
+	background: none;
 	text-decoration: none;
 }
 
 /*Hover state for top level links*/
 li:hover a {
-	background: #19c589;
+	color:  #e12b00;
+  border-bottom-style: solid;
```
Comment (akshatnitd) [inline _includes/header.html]: Please correct the indentation.

File: _includes/header.html [hunk 6]
```diff
@@ -217,33 +220,39 @@
...
 
 /*Style for dropdown links*/
 li:hover ul a {
-	background: #f3f3f3;
-	color: #2f3036;
+	background: #00000080;
+	color: #fff;
 	height: 40px;
 	line-height: 40px;
+  border: none;
```
Comment (akshatnitd) [inline _includes/header.html]: Please correct the indentation.

File: _includes/header.html [hunk 7]
```diff
@@ -310,6 +318,7 @@
 	width: 65%;
 	float: left;
 	height: 285em;
+  padding: 15px;
```
Comment (akshatnitd) [inline _includes/header.html]: Please correct the indentation.

**Review by akshatnitd — CHANGES_REQUESTED** [review]
Please check indentation issues. Rest all looks good for this PR. Also, you can open up issues after it gets merged for other design issues, like changing background color of drop-down in header buttons, etc.

**Comment by akshatnitd** [comment]
Also, you can remove these errors by moving to `https`
screenshot from 2017-12-27 19-25-04

**Review by ParthS007 — CHANGES_REQUESTED** [review]
@Ploypiti Please do the suggested change

**Comment by Ploypiti** [comment]
@akshatnitd @ParthS007 Fixed.

**Comment by Ploypiti** [comment]
Btw should we remove the `_site` file since there's no use of it..? I mean everything are in .md file *outside* and CSS changes made in `_site` doesn't work.

**Review by ParthS007 — APPROVED** [review]
Looks cool 👍

**Comment by Ploypiti** [comment]
@ParthS007 Please merge.

**Review by akshatnitd — CHANGES_REQUESTED** [review]
There still are indentation issues. Please fix those!

**Comment by Ploypiti** [comment]
@akshatnitd Fixed (Hope this is the last time..?).

File: _includes/header.html [hunk 8]
```diff
@@ -217,33 +220,39 @@
 /*Style for menu links*/
 li a {
 	display:block;
+  font-size: 12px;
+  letter-spacing: 1px;
```
Comment (akshatnitd) [inline _includes/header.html]: See indentation here

**Review by akshatnitd — CHANGES_REQUESTED** [review]
Check the indentation issues, where I had pointed out in my first review. All the changes are pending. I guess you are having problems due to tabs and spaces.Check that once.

**Review by akshatnitd — APPROVED** [review]
Looks good now :) 
Happy to see the patience growing with time :100: 

Now, you can open up issues for other design issues, like changing background color of drop-down in header buttons, etc.

**Comment by kubami9** [comment]
Excuse me, I know it's not really a good place to ask but fossasia gitter chat is down and I'm really stuck. I have a problem with `jekyll serve` command, after running, automatically more than 50 files are changed/added/deleted even though I don't touch anything. 
Is there any place, chatroom etc. where I can find help? Sorry for bothering 😅

**Comment by nzec** [comment]
@kubami9 Use the new FOSSASIA gitter chat. (It is a private channel.)

**Comment by kubami9** [comment]
@duttaditya18 Could you give me the link?

**Comment by nzec** [comment]
I added you to the room. Please check.

**Pull request merged**